tests/unit/backend/modules/bounce/bounce.controller.spec.ts

```diff
@@ -0,0 +1,59 @@
+import HttpStatus from 'http-status-codes'
+import { mocked } from 'ts-jest/utils'
+
+import handleSns from 'src/app/modules/bounce/bounce.controller'
+import * as bounceService from 'src/app/modules/bounce/bounce.service'
+
+jest.mock('src/app/modules/bounce/bounce.service')
+const mockBounceService = mocked(bounceService, true)
+
+describe('handleSns', () => {
+  let req, res
+  beforeEach(() => {
+    req = { body: 'somebody' }
+    res = { sendStatus: jest.fn() }
```

Comment:
Should we use `mockResponse` and `mockRequest` exported from our `jest-express` helper files? So the shape is as expected. 

`req` can also be a constant since it does not change, don't need to be in a `beforeEach` block

For response, can we inline it in tests since that is the part of the testable criteria? Reduces the chances of the mock still retaining a previous tests' values since we do not reset them.

e.g.

```ts
it('should run some test', async () => {
      // Arrange
      const mockRes = expressHandler.mockResponse()
      ...
})
```

Reply:
nit: also rename `req` to `MOCK_REQ` to make it obvious it is mock, and is a constant that is never changed